Autopilot/Simulation/SimDriver/main.cpp

```diff
@@ -0,0 +1,29 @@
+
+#include "attitudeManager.hpp"
+
+#include "SendInstructionsToSafety.hpp"
+
+/***********************************************************************************************************************
+ * Code
+ **********************************************************************************************************************/
+
+#define NUM_SIMULATION_STEPS 1000
```

Comment:
By my understanding, this is the number of times that the autopilot will return a packet of data (contains aircraft position, attitude, etc. [defined in WARG Docs]).

If this is the case, approximately how long will a simulation with 1000 steps be?

Reply:
Actually, this is the number of times Autopilot sends new actuator commands to the safety chip. That frequency is something we haven't chosen yet (though 50-200 Hz is common). 

In Simulink, I set the value to 50 Hz, which means the model updates considers that there are 20 ms intervals between each "Step". 

So 1000 steps gets us 20 seconds of simulated flight.